src/tests/ftest/util/telemetry_utils.py

```diff
@@ -449,6 +449,29 @@ def get_container_metrics(self):
                         data[host][name] = metric["value"]
         return data
 
+    def get_io_metrics(self, test_metrics=None):
+        """Get the telemetry test io metrics.
+
```

Comment:
Need to add a little description for `test_metrics` in docstring

Reply:
updated.